test/functional/test_runner.py

```diff
@@ -613,14 +613,12 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
     max_len_name = len(max(test_list, key=len))
     test_count = len(test_list)
```

Comment:
Maybe we can remove `test_count`? See below.

Reply:
`test_count` being assigned outside of the loop enables reporting of the correct total number of total tests on the right side of the `/`.